project/src/components/app/app.tsx

```diff
@@ -1,5 +1,13 @@
-function App(): JSX.Element {
-  return <p>Hello, world!</p>;
+import StartScreen from '../../pages/start-screen/start-screen';
+
+type AppScreenProps = {
+  cardsCount: number;
+};
+
+function App({cardsCount}: AppScreenProps): JSX.Element {
+  return (
+    <StartScreen cardsCount = {cardsCount} />
```

Comment:
Форматируем только так 
```suggestion
    <StartScreen cardsCount={cardsCount} />
```